gensim/models/ldamodel.py

```diff
@@ -367,7 +367,15 @@ def inference(self, chunk, collect_sstats=False):
         # Lee&Seung trick which speeds things up by an order of magnitude, compared
         # to Blei's original LDA-C code, cool!).
         for d, doc in enumerate(chunk):
-            ids = [id for id, _ in doc]
+            if isinstance(doc, numpy.ndarray):
+                if len(doc) == 0:
```

Comment:
`len(array)` will return the length along the first axis (=number of rows). Is that always the case here?